StackableGems/mod.js

```diff
@@ -151,26 +151,59 @@ for (let i = 0; i < ITEM_TYPES.length; i = i + 1) {
 
 // this is behind a config option because it's *a lot* of recipes...
 if (config.convertWhenDestacking) {
-  for (let i = 0; i < ITEM_TYPES.length; i = i + 1) {
-    const itemtype = ITEM_TYPES[i];
-    const stacktype = converItemTypeToStackItemType(itemtype);
-    for (let j = 2; j <= 500; j = j + 1) {
+  for (let i = 0; i < ITEM_TYPES.length - 1; i = i + 1) {
+    const itemtype_cur = ITEM_TYPES[i];
+    const stacktype_cur = converItemTypeToStackItemType(itemtype_cur);
+    const itemtype_up = ITEM_TYPES[i + 1];
+    const stacktype_up = converItemTypeToStackItemType(itemtype_up);
```

Comment:
This should follow pre-existing lowerCamelCase naming convention.
```suggestion
    const itemtype = ITEM_TYPES[i];
    const stacktype = converItemTypeToStackItemType(itemtype_cur);
    const itemtypeUpgraded = ITEM_TYPES[i + 1];
    const stacktypeUpgraded = converItemTypeToStackItemType(itemtype_up);
```